adaptivefiltering/dataset.py

```diff
@@ -38,6 +41,80 @@ def __init__(self, filename, warning_threshold=750000):
                 )
             )
 
+    def generate_geoTif(
+        self,
+        filename,
+        resolution=2.0,
+    ):
+        """Calculate and safe a meshgrid of the dataset with a custom resolution.
+
+        :param filename:
+            Filename to save the .tif. This can be a relative or absolute path
+        :type filename: str
+        :param resolution:
+            The resolution used to calculate the mesh from the point cloud.
```

Comment:
The units of this resolution should be clarified and IMO should be in meters. Not sure whether we need to do some type of conversion with the Lidar data to get there.

Reply:
As far as I can tell from the `.GetProjectionRef()` function of the gdal package the resolution is in meters.
I added this information to the docstring.